### DISCUSSION TITLE
how to save hasMany

### DISCUSSION THREAD

**Discussion by LordVersA** [adonisjs/core#1987] [Help] [answered]
how can i save hasmany in adonis v5? 
i have 2 models: Product and Category
```
newProduct.category_id = productDetails.category;
await newProduct.save();
```
it gives this error:
`insert into products (category_id, created_at, description, image, price, title, total_count, updated_at) values (1, '2020-11-30 20:18:06', 'test', 'files/2020/11/30/1606754880770.png', 14000, 'test product', 0, '2020-11-30 20:18:06') - ER_BAD_FIELD_ERROR: Unknown column 'category_id' in 'field list'(edited)`

**Comment by thetutlage** [comment, marked as answer]
Follow the docs please. https://preview.adonisjs.com/guides/model-relations/has-many#persisting-relationship
